gadgets/Makefile

```diff
@@ -83,6 +83,15 @@ $(GADGETS_README):
 	@# Replace flags section
 	sed -i '/^## Flags/,/^## /{/^## /!d}' $@
 	sed -i '/## Flags/r '<(gomplate -d gadget=$(@:README.mdx=gadget.yaml) -d artifacthubpkg=$(@:README.mdx=artifacthub-pkg.yml) --file README-flags.template) $@
+	@# Replace architecture section and insert mermaid graph
+	sed -i '/^## Architecture/,/^## /{/^## /!d}' $@
+	echo -e "\n\n### Flowchart Graph\n\nMermaid graph showing relations between maps and programs\n\`\`\`mermaid" >> $@
+	sudo ig image inspect $(@:/README.mdx=):$(GADGET_TAG) -o custom --extra-info=ebpf.mermaid.flowchart >> $@
+	echo "\`\`\`" >> $@
+	echo -e "\n### Sequence Graph \n\n" >> $@
+	echo -e "Mermaid graph showing the sequence of events\n\`\`\`mermaid" >> $@
+	sudo ig image inspect $(@:/README.mdx=):$(GADGET_TAG) -o custom --extra-info=ebpf.mermaid.sequence >> $@
+	echo -e "\n\`\`\`" >> $@
```

Comment:
Could you run this on the existing gadgets in the same PR, so we can see if it produces a good output?